docs/tutorial_digital_earth_pacific_patch_level.ipynb

```diff
@@ -0,0 +1,1311 @@
+      "text/plain": [
+       "<Figure size 400x300 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Extract coordinate system from first item\n",
+    "epsg = items[0].properties[\"proj:epsg\"]\n",
+    "\n",
+    "# Convert point from lon/lat to UTM projection\n",
+    "poidf = gpd.GeoDataFrame(crs=\"OGC:CRS84\", geometry=[area_of_interest.centroid]).to_crs(\n",
+    "    epsg\n",
+    ")\n",
+    "geom = poidf.iloc[0].geometry\n",
+    "\n",
+    "# Create bounds of the correct size, the model\n",
+    "# requires 512x512 pixels at 10m resolution.\n",
+    "bounds = (geom.x - 2560, geom.y - 2560, geom.x + 2560, geom.y + 2560)\n",
+    "\n",
+    "# Retrieve the pixel values, for the bounding box in\n",
+    "# the target projection. In this example we use only\n",
+    "# the RGB and NIR band groups.\n",
+    "stack = stackstac.stack(\n",
+    "    items,\n",
+    "    bounds=bounds,\n",
+    "    snap_bounds=False,\n",
+    "    epsg=epsg,\n",
+    "    resolution=10,\n",
+    "    dtype=\"float32\",\n",
+    "    rescale=False,\n",
+    "    fill_value=0,\n",
+    "    assets=BAND_GROUPS[\"rgb\"] + BAND_GROUPS[\"nir\"],\n",
+    "    resampling=Resampling.nearest,\n",
```

Comment:
Since we're working with optical imgaery, it is recommend to set pixel registation to center registered, rather than `stackstac`'s default topleft (gridline) registration. See https://stackstac.readthedocs.io/en/latest/api/main/stackstac.stack.html#stackstac.stack.params.xy_coords.

```suggestion
    "    resampling=Resampling.nearest,\n",
    "    xy_coords=\"center\",\n",
```